### DISCUSSION TITLE
Ring plugin failed after laest update 13.0.1

### DISCUSSION THREAD

**Discussion by thermos22** [dgreif/ring#1472] [General]
The log file says the below, how can I fix.

Loaded plugin 'homebridge-ring'
8/19/2024, 7:35:52 PMRing BridgeLoading 1 platforms...
8/19/2024, 7:35:52 PMRing BridgeRingConfiguring cached accessory a36a94c0-bf03-4b03-957c-93d4ae572bbf Front Door
8/19/2024, 7:35:52 PMRing BridgeRingConfiguring cached accessory c1b14ba1-2fbd-479e-927f-b857c18d7579 Backyard
8/19/2024, 7:35:52 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:35:52 PMRing BridgeBridge is running on port 51826.
8/19/2024, 7:35:57 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:35:57 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:36:02 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:36:02 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:36:07 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:36:07 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:36:12 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...
8/19/2024, 7:36:12 PMRing BridgeRingERRORFailed to reach Ring server at https://oauth.ring.com/oauth/token.  fetch is not defined.  Trying again in 5 seconds...

**Comment by tsightler** [comment]
> The log file says the below, how can I fix.

Upgrade your NodeJS to a supported version (at least v18, ideally v20).  The release notes of the v13 plugin list the requirement for NodeJS >=v18 and this is also the currently supported versions for latest Homebridge as well.  If on HOOBS, attempt to contact HOOBS support, only they can help.  

This is a duplicate of https://github.com/dgreif/ring/issues/1459, https://github.com/dgreif/ring/issues/1463, and  https://github.com/dgreif/ring/issues/1469.  To save time, we request that users leverage search before posting new issues.

**Comment by thermos22** [comment]
How do I do that? I’m new to all this. Thank you. Sent from my iPhoneOn Aug 19, 2024, at 7:56 PM, tsightler ***@***.***> wrote:﻿

The log file says the below, how can I fix.

Upgrade your NodeJS to a supported version (at least v18, ideally v20).  The release notes of the v13 plugin list the requirement for NodeJS >=v18 and this is also the currently supported versions for latest Homebridge as well.  If on HOOBS, attempt to contact HOOBS support, only they can help.
This is a duplicate of #1459, #1463, and  #1469.  To save time, we request that users leverage search before posting new issues.

**Reply by tsightler** [reply]
This is a Homebridge question, not a question for plugin authors, and the steps will vary based on how you are running Homebridge, but this should help:

https://github.com/homebridge/homebridge/wiki/How-To-Update-Node.js/

If you are using HOOBS, then contact HOOBS support, they are the only ones that can help you.
